hkmc2/shared/src/test/mlscript/codegen/Getters.mls

```diff
@@ -276,7 +277,9 @@ class Foo(x) with
   fun oops = x
 //│ JS (unsanitized):
 //│ let Foo1;
-//│ Foo1 = function Foo(x1) { return new Foo.class(x1); };
+//│ Foo1 = function Foo(x1) {
+//│   return new Foo.class(x1);
+//│ };
```

Comment:
Any idea why there are no longer on one line?

Reply:
When there's parameter lists, we need to curry the ctor using lambdas, and it's quite ugly to have it all on one line. I decided it was not worth special-casing non-curried ctors to be on one line such it's purely a superficial change.

Reply:
Oh I see, I didn't realize the line was previously missing the breaks.